### PR TITLE
LPS-116528

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://issues.liferay.com/browse/LPS-116528

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/util/evaluation.es.js [hunk 1]
```diff
@@ -117,10 +117,16 @@ export const mergePages = (
...
-				) || {};
+			let sourceField =
+				sourcePagesVisitor.findField(({name}) => name === field.name) ||
+				{};
+
+			if (!sourceField || Object.keys(sourceField).length === 0) {
+				sourceField =
+					sourcePagesVisitor.findField(
+						({fieldName}) => fieldName === field.fieldName
+					) || {};
```
Comment (oliveiraaraujo) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/util/evaluation.es.js:128]: @carolmariaabb What do you think using "nullish coalescing operator" (??) here instead "||" operator ?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldSet/FieldSet.es.js [hunk 2]
```diff
@@ -38,14 +38,22 @@ const getRows = (rows, nestedFields) => {
...
 		...row,
-		columns: row.columns.map((column) => ({
-			...column,
-			fields: column.fields.map((fieldName) => {
-				return nestedFields.find(
-					(nestedField) => nestedField.fieldName === fieldName
+		columns: row.columns.map((column) => {
+			let fields = [];
+
+			column.fields.forEach((fieldName) => {
```
Comment (oliveiraaraujo) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldSet/FieldSet.es.js:44]: @carolmariaabb , what do you think about using a array.reduce here instead a forEach ?
eg. 
```
let fields = column.fields.reduce(...., []);
```

**Comment by jeyvison** [comment]
ci:forward

**Pull request closed without merging**